pom.xml

```diff
@@ -1353,15 +1363,15 @@
     <profile>
       <id>cdh5</id>
       <activation>
+        <activeByDefault>true</activeByDefault>
         <property>
           <name>hadoop.profile</name>
           <value>cdh5</value>
         </property>
       </activation>
-      <properties>
-        <!-- test cdh5 with the same jars that were produced for default profile -->
-        <maven.main.skip>true</maven.main.skip>
-      </properties>
+      <modules>
+        <module>kite-maven-plugin</module>
```

Comment:
Will we ever need this module for HDP?
I assume no, but perhaps this setup is unusual enough to justify a quick line of documentation?